github_server/action.yml

```
@@ -0,0 +1,53 @@
+name: "ALKiln/github_server: Isolated docassemble tests"
+description: "Automatically test docassemble interviews on GitHub's servers, in an isolated environment. Avoids docassemble reload errors and other external factors"
+
+
+inputs:
+  CONFIG_CONTENTS:
+    description: "The contents of the config file of the docassemble server"
+    required: false
+  # This is necessary to test pre-releases at the very least. It does seem
+  # strange considering we're controlling everything through the action, but
+  # that actually doesn't touch the npm version that's pulled in.
+  ALKILN_VERSION:
+    description: 'Version of ALKiln to get from npm.'
+    required: false
+    default: '^5.0.0'
+  MAX_SECONDS_FOR_SETUP:
+    description: "Amount of time to give the Project to upload your package's GitHub code."
+    required: false
+    default: '120'
+  SERVER_RELOAD_TIMEOUT_SECONDS:
+    description: "Max amount of time to give the server to reload if it needs to"
+    required: false
+    # no default to let the custom Scenario timeouts create the reload timeout
+
+
+runs:
+  using: "composite"
+  steps:
+      - name: "github_server: Set environment variables"
+        run: |
+          echo "DA_ADMIN_API_KEY=abcd1234abcd1234abcd5678abdc5678" >> $GITHUB_ENV
+        shell: bash
+      - name: Download and start docker
+        run: |
+          mkdir /tmp/config
+          echo "$CONFIG_CONTENTS" > /tmp/config/myconfig.yml 
+          docker pull brycewilley/smoldocassemble
```

```suggestion
          docker pull jhpyle/docassemble
```

This will avoid an extra 1-2 minutes of downloading a docker image that we aren't using anymore (though it's worth considering trying that again it keeps taking that long for everything to start up).